macos/xcode/FluentUI_swiftui_testapp.xcconfig

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+#include "FluentUI_common.xcconfig"
+
+MACOSX_DEPLOYMENT_TARGET = 10.16
```

Comment:
Can the test app with SwiftUI run on 10.15?

Reply:
Nope, because I chose to use the SwiftUI App construct. That's 10.16 (11.0) only, but it's what makes this so simple